Watchman.Discord/Areas/UselessFeatures/Controllers/UselessController.cs

```diff
@@ -21,5 +24,13 @@ public async Task PrintMarchew(MarchewCommand command, Contexts contexts)
             var messagesService = this._messagesServiceFactory.Create(contexts);
             await messagesService.SendMessage(text);
         }
+        public async Task SendMarudaImage(MarudaCommand command, Contexts contexts)
+        {
+            var imagesService = new ImagesService();
```

Comment:
to powinno wchodzić przez konstruktor (dependency injection)
pod spodem jest IoC więc stworzy sie automatycznie 

w IoC jest automat który próbuje zarejestrować wszystkie typy jakie się da